src/pages/_app.tsx

```diff
@@ -112,14 +112,16 @@ const App: React.FC<React.PropsWithChildren<AppProps>> = ({
         <ViewerProvider>
           <LogRocketProvider>
             <CioProvider>
-              <QueryClientProvider client={queryClient}>
-                <MDXProvider components={mdxComponents}>
-                  {getLayout(Component, pageProps)}
-                </MDXProvider>
-                <div className="print:hidden">
-                  <ReactQueryDevtools />
-                </div>
-              </QueryClientProvider>
+              <TrpcProvider>
+                <QueryClientProvider client={queryClient}>
+                  <MDXProvider components={mdxComponents}>
+                    {getLayout(Component, pageProps)}
+                  </MDXProvider>
+                  <div className="print:hidden">
+                    <ReactQueryDevtools />
+                  </div>
+                </QueryClientProvider>
+              </TrpcProvider>
```

Comment:
wrapping app with provider instead of using `withTRPC` that comes from trpc/next